daemon/core.h

```diff
@@ -100,52 +100,52 @@ class Core : public QThread, public LogTarget
     using ClientPaths           = QSet<ClientPath>;
     using ClientPathsBySender   = QMap<QString, ClientPaths>;
 
-private slots:
-    void serviceDisappeared(const QString &sender);
+private:
+    Q_INVOKABLE void serviceDisappeared(const QString &sender);
```

Comment:
Do we even need this methods to be available to Qt meta system when we're using the modern (Qt5) signal/slot connection?

Reply:
Good question! Somehow didn't realize it is possible now to connect function pointers directly. Thanks!